### ISSUE TITLE
Analysis Errors Should Log to Stdout

### ISSUE THREAD

**Issue by ax3l** [AMReX-Codes/regression_testing#96]
In local tests, I quite often forget to activate the right virtual environment or have just forgotten to install a few Python modules in a fresh virtual env. As a consequence, the Python scripts that we use for testing fail, e.g., with an import error.

This error fails the test that ran it, but no info besides "<testname> FAILED" is printed to stdout. After some analysis, I then find that `{output_dir}/{test_name}.analysis.out` contains some error message.

Long story short: we should log a line when the analysis step returns a non-zero error code. This would then be same way as we also report compile errors or crashes and speeds up error diagnosis for mortal humans like me.

**Comment by ax3l** [comment]
Added in #114
